### PR TITLE
added serge file

### PR DESCRIPTION
**Pull request by iFun**

### CODE REVIEW

File: edit-in-place.serge.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      "ja-jp ja",
+      "ko-kr ko",
+      "nl-nl nl",
+      "pt-br pt",
+      "sv-se sv",
+      "tr-tr tr",
+      "zh-cn zh",
+      "zh-tw zh-tw"
+    ]
+  }
```
Comment (m6jones) [inline edit-in-place.serge.json]: Would you be able to add an owner that will get added to the translation PRs. You, Brad and myself can be added.

Comment (iFun) [reply edit-in-place.serge.json]: where should I add the owner? is it comment in this file?

Comment (AKobets) [reply edit-in-place.serge.json]: @BhamelinD2L  Is it the https://github.com/BrightspaceUILabs/edit-in-place/blob/master/CODEOWNERS that shoul be updated ?

Comment (BhamelinD2L) [reply edit-in-place.serge.json]: @m6jones Is almost definitely referring to the codeowners file, yes.
You should be able to add our names to the file and remove the `pr_reviewers` section from serge.
(drop the extra @ in front of my username in that file to add me, that isn't correct and got added during the template's generation script)
Once that's done, we'll be notified whenever a PR is opened for the repo.

**Comment by iFun** [comment]
> Looks like everything's in order to me. 👍

Do i need to make a new release or just wait for the translation PR then increase

**Pull request merged**